Flipper/Packages/Core/Sources/Storage/Patform/FileStorage+Codable.swift

```diff
@@ -13,11 +13,7 @@ extension FileStorage {
 
 extension FileStorage {
     func read<T: Codable>(_ path: Path) throws -> T? {
-        let string = try read(path)
-        guard let data = string.data(using: .utf8) else {
-            return nil
-        }
-        return try? JSONDecoder().decode(T.self, from: data)
+        return try? JSONDecoder().decode(T.self, from: read(path))
```

Comment:
`Codable`s some times decoding some JSON that might come with a different and unexpected key, can throw an error that is interesting to be caught using a do/catch ;)